codespell_lib/data/dictionary.txt

```diff
@@ -8752,6 +8755,7 @@ finishs->finishes
 finitel->finite
 finness->finesse
 finnished->finished
+finshed->finished
```

Comment:
Add finsh and finshes (also suggest finch/finches for these)

Reply:
744ce7e